src/renderer/state/types.ts

```diff
@@ -287,6 +287,7 @@ export interface EnabledNotifications {
 
 export interface SettingStateBranch extends LimsUrl {
   metadataColumns: string[];
+  showAddAnnotationHint: boolean;
```

Comment:
There doesn't appear to be a way to mark this as false, so it seems it is perpetually true. Regardless, the way the `showUploadHint` and `showTemplateHint` are toggled is within the settings modal and I think this "add annotation" hint should be considered a template hint and controlled by that variable rather than as an additional more niche setting

Reply:
I didn't really understand what the settings were for -- fixed